frontend/src/features/frame/components/sidebar/Footer.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+
+import { handleWikiUrl } from 'src/utils/url';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    padding: 8,
+    borderTop: '1px solid #806300',
```

Comment:
personal preference

The color `#806300` is quite strong, I would have used the color `#e7e5db` suggested by Figma instead.

Reply:
Improved!